pkg/server/httpAPI.go

```diff
@@ -43,6 +44,8 @@ const (
 	initializationStatusProgress   = "Progress"
 	initializationStatusSuccessful = "Successful"
 	initializationStatusFailed     = "Failed"
+
+	serverReadHeaderTimeout = 5 * time.Second
```

Comment:
can you also add 1 line of comment why this ReadHeaderTimeout is required ? 
It will be helpful for future reference.

Reply:
Done